src/sentry/static/sentry/app/views/releasesV2/list/releasehealth/compactContent.tsx

```diff
@@ -0,0 +1,75 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import {PanelBody} from 'app/components/panels';
+import {ReleaseProject, Release} from 'app/types';
+import space from 'app/styles/space';
+
+import ClippedHealthRows from '../clippedHealthRows';
+import Header from './header';
+import Item from './item';
+import ProjectName from './projectName';
+import IssuesQuantity from './issuesQuantity';
+
+type Props = {
+  projects: Array<ReleaseProject>;
+  releaseVersion: Release['version'];
+  orgSlug: string;
+};
+
+const CompactContent = ({projects, releaseVersion, orgSlug}: Props) => (
+  <React.Fragment>
+    <Header>{t('Projects')}</Header>
+    <PanelBody>
+      <StyledClippedHealthRows
+        fadeHeight="46px"
+        maxVisibleItems={4}
+        columnsQuantity={projects.length}
+      >
+        {projects.map(project => {
+          const {id, slug, newGroups = 0} = project;
+          return (
+            <StyledItem key={`${releaseVersion}-${slug}-health`}>
+              <ProjectName
+                orgSlug={orgSlug}
+                project={project}
+                releaseVersion={releaseVersion}
+              />
+              <IssuesQuantity
+                orgSlug={orgSlug}
+                releaseVersion={releaseVersion}
+                projectId={id}
+                newGroups={newGroups}
+                isCompact
+              />
+            </StyledItem>
+          );
+        })}
+      </StyledClippedHealthRows>
+    </PanelBody>
+  </React.Fragment>
+);
+
+export default CompactContent;
+
+const StyledItem = styled(Item)`
+  align-items: center;
+  border-right: 1px solid ${p => p.theme.borderLight};
+  display: grid;
+  grid-template-columns: 1fr max-content;
+  grid-column-gap: ${space(2)};
+  :last-child {
+    border-right: 1px solid ${p => p.theme.borderLight};
+  }
+`;
+
+const StyledClippedHealthRows = styled(ClippedHealthRows)<{columnsQuantity: number}>`
+  display: grid;
+  grid-template-columns: 1fr;
+  margin-right: -1px;
+  margin-bottom: -1px;
+  @media (min-width: ${p => p.theme.breakpoints[0]}) {
```

Comment:
Would it make sense to show more than two columns on wider screens?

Reply:
I thought we agreed on 2 columns, but yes I can update it 😉

Reply:
Those columns are relatively compact and if release is associated with 40 projects I think on wider screens it would make sense to have it in 3, maybe even 4 columns. Let's sync on the logic.